### PR TITLE
OSDOCS-3704: Adding release notes for secondary scheduler operator 1.0.1

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.10 and 4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3704

Link to docs preview:
http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3704/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html#secondary-scheduler-operator-release-notes-1.0.1

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
Not to be merged until July 28th

**Comment by bergerhoffer** [comment]
@kasturinarra @ingvagabund The SSO 1.0.1 release notes are ready for review. Note that this will be going into **4.10 and 4.11** - if you can please confirm that is correct.

Preview: http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3704/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html#secondary-scheduler-operator-release-notes-1.0.1

File: nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc [hunk 1]
```diff
@@ -12,6 +12,35 @@ These release notes track the development of the {secondary-scheduler-operator-f
...
+
+[id="secondary-scheduler-operator-1.0.1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The maximum {product-title} version for {secondary-scheduler-operator-full} 1.0.1 is 4.11.
+
+[id="secondary-scheduler-1.0.1-bug-fixes"]
+=== Bug fixes
+
+* Previously, the secondary scheduler deployment was not deleted after the secondary scheduler custom resource (CR) was deleted, which caused the {secondary-scheduler-operator} and operand not to be fully uninstalled. The secondary scheduler deployment is now deleted when the secondary scheduler CR is deleted, so that the {secondary-scheduler-operator} can now be fully uninstalled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100923[*BZ#2100923*])
```
Comment (jldohmann) [inline nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:37]: The double negative here is a little hard to parse, so suggestion to change sentence to something like the following:
```suggestion
* Previously, the secondary scheduler deployment was not deleted after the secondary scheduler custom resource (CR) was deleted, which prevented the {secondary-scheduler-operator} and operand from being fully uninstalled. The secondary scheduler deployment is now deleted when the secondary scheduler CR is deleted, so that the {secondary-scheduler-operator} can now be fully uninstalled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100923[*BZ#2100923*])
```

**Review by jldohmann — APPROVED** [review]
LGTM, left a suggestion below

**Comment by ingvagabund** [comment]
LGTM

File: nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:37]: Great suggestion! I wasn't a fan of that wording either, but couldn't think of anything better :)

File: nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc [hunk 2]
```diff
@@ -12,6 +12,35 @@ These release notes track the development of the {secondary-scheduler-operator-f
...
+
+[id="secondary-scheduler-operator-1.0.1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The maximum {product-title} version for {secondary-scheduler-operator-full} 1.0.1 is 4.11.
+
+[id="secondary-scheduler-1.0.1-bug-fixes"]
+=== Bug fixes
+
+* Previously, the secondary scheduler deployment was not deleted after the secondary scheduler custom resource (CR) was deleted, which prevented the {secondary-scheduler-operator} and operand from being fully uninstalled. The secondary scheduler deployment is now deleted when the secondary scheduler CR is deleted, so that the {secondary-scheduler-operator} can now be fully uninstalled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100923[*BZ#2100923*])
```
Comment (kasturinarra) [inline nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:37]: @bergerhoffer I see this is a  4.11 bug here which is fixed in SSO 1.1.0, so i do not think it should be listed here.

Comment (bergerhoffer) [reply nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:37]: Confirmed via slack that this BZ was a bug in 1.0 and fixed in 1.0.1, so is appropriate to list here

**Comment by bergerhoffer** [comment]
SSO 1.0.1 has shipped; publishing RNs.
